### PR TITLE
Mobile v1.55.1 ESR Notes

### PR DESCRIPTION
**Pull request by amyblais**
v1.56.0 release was cancelled.

Given the low volume of changes and to simplify our server <> mobile compatibility guidance approaching the v2 launch, we're going to skip the v1.56 release. That means v1.55.1 (released in September) is the last v1 release.

v1.55 will also be our first ever mobile [ESR](https://handbook.mattermost.com/company/about-mattermost/list-of-terms#esr), supporting server v6.3+.

### CODE REVIEW

**Review by enahum — APPROVED** [review]

**Review by cpoile — APPROVED** [review]

**Review by esethna — APPROVED** [review]

**Comment by amyblais** [comment]
I added some details on v1.56 ESR, so I'm queueing this for re-reviews.

File: source/deploy/mobile-app-changelog.md [hunk 1]
```diff
@@ -2,12 +2,29 @@
...
+- [1.54.0 Release](#id9)
+- [1.53.0 Release](#id14)
+- [1.52.0 Release](#id18)
+
+## 1.56.0 Release
+- Release Date: November 16, 2022
+- Server Versions Supported: Server v7.1.0+ is required. Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device.
+
+### Compatibility
+ - Mobile App v1.56 is our first extended support release (ESR). We are very excited about the upcoming general availability of the v2.0 mobile app in December. We recognize that some customers may have a custom build of Mattermost mobile and need more time to test or implement their custom changes on mobile v2.0. The mobile ESR will be supported until July 2023. v1.56 will receive critical security fixes only and be released as needed via unsigned builds to our mobile GitHub repository.
```
Comment (cwarnermm) [inline source/deploy/mobile-app-changelog.md:17]:
```suggestion
 - Mobile App v1.56 is our first extended support release (ESR). We are very excited about the upcoming general availability of the v2.0 mobile app in December. We recognize that some customers may have a custom build of Mattermost mobile and need more time to test or implement their custom changes on mobile v2.0. The mobile ESR will be supported until July 2023. v1.56 will receive critical security fixes only and will be released as needed via unsigned builds to our mobile GitHub repository.
```

File: source/deploy/mobile-app-changelog.md [hunk 2]
```diff
@@ -2,12 +2,29 @@
...
+ - **Upgrade to server version v7.1.0 or later is required.** Support for server [Extended Support Release](/upgrade/extended-support-release.html) (ESR) 6.3.0 has ended and upgrading to server ESR v7.1.0 or later is required. As we innovate and offer newer versions of our mobile apps, we maintain backwards compatibility only with supported server versions. Users who upgrade to the newest mobile apps while being connected to an unsupported server version can be exposed to compatibility issues, which can cause crashes or severe bugs that break core functionality of the app.
```
Comment (cwarnermm) [inline source/deploy/mobile-app-changelog.md:18]:
```suggestion
 - **Upgrade to Mattermost server version v7.1.0 or later is required.** Support for server [Extended Support Release](/upgrade/extended-support-release.html) (ESR) 6.3.0 has ended and upgrading to server ESR v7.1.0 or later is required. As we innovate and offer newer versions of our mobile apps, we maintain backwards compatibility only with supported server versions. Users who upgrade to the newest mobile apps while being connected to an unsupported server version can be exposed to compatibility issues, which can cause crashes or severe bugs that break core functionality of the app.
```

**Review by cwarnermm — APPROVED** [review]

**Comment by cwarnermm** [comment]
I've provided feedback inline. Approving for expediency.

File: source/deploy/mobile-app-changelog.md [hunk 3]
```diff
@@ -2,12 +2,29 @@
...
+ - **Upgrade to Mattermost server version v7.1.0 or later is required.** Support for server [Extended Support Release](/upgrade/extended-support-release.html) (ESR) 6.3.0 has ended and upgrading to server ESR v7.1.0 or later is required. As we innovate and offer newer versions of our mobile apps, we maintain backwards compatibility only with supported server versions. Users who upgrade to the newest mobile apps while being connected to an unsupported server version can be exposed to compatibility issues, which can cause crashes or severe bugs that break core functionality of the app.
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).	
+ - iPhone 5s devices and later with iOS 12.1+ is required.
+
+### Bug Fixes
```
Comment (esethna) [inline source/deploy/mobile-app-changelog.md:22]: Was this the only fix included?

**Review by esethna — APPROVED** [review]

**Comment by esethna** [comment]
@amyblais please work with @nab-77 on any messaging regarding the mobile ESR. Looks fine from my perspective. Thanks!

File: source/deploy/mobile-app-changelog.md (hunk 3 above)
Comment (amyblais) [reply source/deploy/mobile-app-changelog.md:22]: Yes, and there were translations updates and dependency updates.

**Review by esethna — APPROVED** [review]

**Pull request merged**